locale/fr.po

```diff
@@ -0,0 +1,1599 @@
+#: py/modthread.c:240
+msgid "expecting a dict for keyword args"
+msgstr "un dict est attendu pour les arguments nommés"
+
+#: py/moduerrno.c:108 py/moduerrno.c:111
+msgid "Permission denied"
+msgstr "Permission refusée"
+
+#: py/moduerrno.c:109
+msgid "No such file/directory"
+msgstr "Fichier/dossier introuvable"
+
+#: py/moduerrno.c:110
+msgid "Input/output error"
+msgstr "Erreur d'entrée/sortie"
+
+#: py/moduerrno.c:112
+msgid "File exists"
+msgstr "Le fichier existe"
+
+#: py/moduerrno.c:113
+msgid "Unsupported operation"
+msgstr "Opération non supportée"
+
+#: py/moduerrno.c:114
+msgid "Invalid argument"
+msgstr "Argument invalide"
+
+#: py/obj.c:89
+msgid "Traceback (most recent call last):\n"
+msgstr "Trace (appel les plus récents en dernier):\n"
+
+#: py/obj.c:92
+msgid "  File \"%q\", line %d"
+msgstr "  Fichier \"%q\", ligne %d"
+
+#: py/obj.c:94
+msgid "  File \"%q\""
+msgstr "  Fichier \"%q\""
+
+#: py/obj.c:101
+msgid ", in %q\n"
+msgstr ", dans %q\n"
+
+#: py/obj.c:241
+msgid "can't convert to int"
+msgstr "ne peut convertir en entier int"
+
+#: py/obj.c:244
+#, c-format
+msgid "can't convert %s to int"
+msgstr "ne peut convertir %s en entier int"
+
+#: py/obj.c:304
+msgid "can't convert to float"
+msgstr "ne peut convertir en nombre à virgule flottante (float)"
+
+#: py/obj.c:307
+#, c-format
+msgid "can't convert %s to float"
+msgstr "ne peut convertir %s en nombre à virgule flottante (float)"
+
+#: py/obj.c:337
+msgid "can't convert to complex"
+msgstr "ne peut convertir en nombre complexe"
+
+#: py/obj.c:340
+#, c-format
+msgid "can't convert %s to complex"
+msgstr "ne peut convertir %s en nombre complexe"
+
+#: py/obj.c:355
+msgid "expected tuple/list"
+msgstr "un tuple ou une liste est attendu"
+
+#: py/obj.c:358
+#, c-format
+msgid "object '%s' is not a tuple or list"
+msgstr "l'objet '%s' n'est pas un tuple ou une liste"
+
+#: py/obj.c:369
+msgid "tuple/list has wrong length"
+msgstr "tuple/liste a une mauvaise longueur"
+
+#: py/obj.c:371
+#, c-format
+msgid "requested length %d but object has length %d"
+msgstr "la longueur requise est %d mais l'objet est long de %d"
+
+#: py/obj.c:384
+msgid "indices must be integers"
+msgstr "les indices doivent être des entiers"
+
+#: py/obj.c:387
+msgid "%q indices must be integers, not %s"
+msgstr "les indices %q doivent être des entiers, pas %s"
+
+#: py/obj.c:407
+msgid "%q index out of range"
+msgstr "index %q hors gamme"
+
+#: py/obj.c:439
+msgid "object has no len"
+msgstr "l'objet n'a pas de len"
+
+#: py/obj.c:442
+#, c-format
+msgid "object of type '%s' has no len()"
+msgstr "l'objet de type '%s' n'a pas de len()"
+
+#: py/obj.c:480
+msgid "object does not support item deletion"
+msgstr "l'objet ne supporte pas la suppression d'éléments"
+
+#: py/obj.c:483
+#, c-format
+msgid "'%s' object does not support item deletion"
+msgstr "l'objet '%s' ne supporte pas la suppression d'éléments"
+
+#: py/obj.c:487
+msgid "object is not subscriptable"
+msgstr "l'objet n'est souscrivable"
```

Comment:
Typo: "l'objet n'est pas souscrivable", and I'm not sure about souscrivable ...

Reply:
"sous-scriptable" should be better